jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ShutdownHandler.java

```diff
@@ -111,9 +111,11 @@ public ShutdownHandler(String shutdownToken, boolean exitJVM)
      * @param shutdownPath the path to respond to shutdown requests against (default is "{@code /shutdown}")
      * @param shutdownToken a secret password to avoid unauthorized shutdown attempts
      * @param exitJVM If true, when the shutdown is executed, the handler class System.exit()
+     * @param handler the handler to wrap
      */
-    public ShutdownHandler(String shutdownPath, String shutdownToken, boolean exitJVM)
+    public ShutdownHandler(String shutdownPath, String shutdownToken, boolean exitJVM, Handler handler)
```

Comment:
I'd like `Handler` as first parameter better.

Reply:
I've put that parameter at the end because `MinimumDataRateHandler` already accepts the handler to wrap as the last ctor parameter.

But I'm not against changing this ordering.